Makefile

```diff
@@ -1,8 +1,50 @@
+# This is the format of an AWS ECR Public Repo as an example.
+export KWOK_REPO ?= ${ACCOUNT_ID}.dkr.ecr.${DEFAULT_REGION}.amazonaws.com
```

Comment:
Should we pushing to an AWS repo since this image should exist out in the open? We should probably talk to SIG Autoscaling or other release SIGs around where we should be storing the artifacts that are published from this repo

Reply:
There's a shorter-term question that I have here which is: Can we just releae the code for the provider and allow users to do a `make run` which runs the binary against their repo but doesn't install it directly as a deployment on their cluster and then do the release piece as a fast-follow once we decide where the artifacts land

Reply:
This was purely for development purposes as an example, not an actual repo where we're storing/publishing artifacts. Users should be able to run `make run` or `make apply` to fit however they want to test upstream Karpenter. Not convinced about the value of doing a release on this though.

Reply:
Got it. Makes sense.